src/puppeteer.js

```diff
@@ -189,6 +193,7 @@ export const launchPuppeteer = async (options = {}) => {
     checkParamOrThrow(options.proxyUrl, 'options.proxyUrl', 'Maybe String');
     checkParamOrThrow(options.useApifyProxy, 'options.useApifyProxy', 'Maybe Boolean');
     checkParamOrThrow(options.puppeteerModule, 'options.puppeteerModule', 'Maybe String');
+    checkParamOrThrow(options.stealth, 'options.stealth', 'Maybe Object');
```

Comment:
In comments it's called `stealthOptions`, here just `stealth`.

Reply:
BTW there should be some simple way to enable all stealth options, otherwise the usage it's just too painful. How about adding a parameter `stealth: Boolean` and having `stealthOptions` just to override the defaults, which are `true` most of the time?

Reply:
That seems like a good idea.